docs/uptime-guide/security.asciidoc

```diff
@@ -52,7 +41,8 @@ PUT /_security/role/uptime
 [float]
 === Assign the role to a user
 
-Next, you'll need to create a user with both the `kibana_user`, and `uptime` roles. 
+Next, you'll need to create a user with both the `uptime` role, and another role with sufficient <<kibana-privileges, Kibana privileges>>,
```

Comment:
Since the page you're linking to is not part of the same book (i.e. https://www.elastic.co/guide/en/kibana/master/kibana-privileges.html), it should be changed to an external link:
```suggestion
Next, you'll need to create a user with both the `uptime` role, and another role with sufficient {kibana-ref}/kibana-privileges.html[Kibana privileges],
```

Reply:
Thanks @lcawl!